db/schema.rb

```diff
@@ -1928,19 +1929,14 @@
     t.integer "version"
   end
 
-  create_table "workflows", id: :integer, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci ROW_FORMAT=DYNAMIC", force: :cascade do |t|
+  create_table "workflows", id: :integer, charset: "utf8mb4", collation: "utf8mb4_unicode_ci", options: "ENGINE=InnoDB ROW_FORMAT=DYNAMIC", force: :cascade do |t|
     t.string "name"
     t.integer "item_limit"
     t.text "locale", size: :medium
     t.integer "pipeline_id"
     t.index ["pipeline_id"], name: "index_workflows_on_pipeline_id"
   end
 
-  add_foreign_key "aliquot_indices", "aliquots", name: "fk_aliquot_indices_to_aliquots"
-  add_foreign_key "aliquot_indices", "receptacles", column: "lane_id"
-  add_foreign_key "aliquots", "primer_panels"
-  add_foreign_key "aliquots", "requests"
-  add_foreign_key "assets_deprecated", "plate_types", column: "labware_type_id"
```

Comment:
I can't see equivalent statements in create_table blocks. Why are these removed?

Reply:
This file is being automatically updated; maybe I should not include it in the PR?

Reply:
db:migrate should handle the updates of the schema. I asked to check if you edited the file directly actually.

Reply:
nope, I did not, this is been updated automatically.

Reply:
nope! I did not have to that